data/rayben109.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Raymond Luanda",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Software Developer @Codeeit, In love with Javascript | React | React Native | Nodejs | Robotics | IoT",
+  "avatar": "https://avatars.githubusercontent.com/u/18146963?v=4",
```

Comment:
I would suggesting using the following for your avatar:

https://github.com/rayben109.png